code/modules/mod/mod_theme.dm

```diff
@@ -1159,6 +1159,88 @@
 	acid = 100
 	wound = 25
 
+/datum/mod_theme/interdyne
+	name = "interdyne"
+	desc = "A corpse-snatching and rapid-retrieval modsuit, resulting from a lucrative tech exchange between Intrudyne Pharmaceutics and Cybersun Industries."
```

Comment:
```suggestion
	desc = "A corpse-snatching and rapid-retrieval modsuit, resulting from a lucrative tech exchange between Interdyne Pharmaceutics and Cybersun Industries."
```

Reply:
thank you for all your grammar checks - misspelling Interdyne as Intrudyne is one I corrected myself on quite often. now they all should be fixed.